<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>867</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="Canvas" name="canvas" native="true">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:white;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="vlConfig">
      <item>
       <widget class="QGroupBox" name="gbPointsRect">
        <property name="minimumSize">
         <size>
          <width>360</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>360</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Координаты точек прямоугольника</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="l_p1">
           <property name="text">
            <string>Координаты первой точки</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSpinBox" name="sb_p1_x">
           <property name="minimum">
            <number>-2000</number>
           </property>
           <property name="maximum">
            <number>2000</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="sb_p1_y"/>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="l_p2">
           <property name="text">
            <string>Координаты второй точки</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QSpinBox" name="sb_p2_x">
           <property name="maximum">
            <number>2000</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="sb_p2_y">
           <property name="maximum">
            <number>2000</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="l_angle">
           <property name="text">
            <string>Угол</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QDoubleSpinBox" name="dsb_angle">
           <property name="minimum">
            <double>-500.000000000000000</double>
           </property>
           <property name="maximum">
            <double>500.990000000000009</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbPointsSet">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>360</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>360</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Координаты точек треугольников</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableWidget" name="twPointsSet">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="showGrid">
            <bool>true</bool>
           </property>
           <property name="gridStyle">
            <enum>Qt::SolidLine</enum>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
           <property name="rowCount">
            <number>4</number>
           </property>
           <property name="columnCount">
            <number>2</number>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>120</number>
           </attribute>
           <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <row/>
           <row/>
           <row/>
           <row/>
           <column>
            <property name="text">
             <string>X</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Y</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string>50</string>
            </property>
           </item>
           <item row="0" column="1">
            <property name="text">
             <string>50</string>
            </property>
           </item>
           <item row="1" column="0">
            <property name="text">
             <string>150</string>
            </property>
           </item>
           <item row="1" column="1">
            <property name="text">
             <string>50</string>
            </property>
           </item>
           <item row="2" column="0">
            <property name="text">
             <string>100</string>
            </property>
           </item>
           <item row="2" column="1">
            <property name="text">
             <string>150</string>
            </property>
           </item>
           <item row="3" column="0">
            <property name="text">
             <string>100</string>
            </property>
           </item>
           <item row="3" column="1">
            <property name="text">
             <string>20</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbPointCount">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum">
            <number>3</number>
           </property>
           <property name="maximum">
            <number>20</number>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbGo">
        <property name="minimumSize">
         <size>
          <width>360</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>360</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Вычислить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>886</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionTask"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionTask">
   <property name="text">
    <string>Условие</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О приложении</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header>canvas.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
